charts/oes/templates/pipeline-promotion/spin-cli-config-secret.yaml

```diff
@@ -10,5 +10,5 @@ stringData:
 kind: Secret
 metadata:
   name: spin-cli-config
-  namespace: {{ .Values.spinnaker.gitopsHalyard.pipelinePromotion.executionNamespace }}
+  #namespace: {{ .Values.spinnaker.gitopsHalyard.pipelinePromotion.executionNamespace }}
```

Comment:
Let's remove this namespace section and let pipeline promotion stuff run in the same namespace as OES. Would it have any side effects?